Makefile

```diff
@@ -19,6 +19,8 @@ GOTEST_BASE = -test.v -check.vv -timeout 360s
 GOTEST_PRIV_OPTS = $(GOTEST_BASE) -tags=privileged_tests
 GOTEST_COVER_OPTS = -coverprofile=coverage.out -covermode=count -coverpkg $(COVERPKG)
 
+JOB_BASE_NAME ?= cilium_test
```

Comment:
should this hint that it's the fallback? Or is the intention that we only set JOB_BASE_NAME only in some cases?

Reply:
We set this in the Jenkins file, and without this the target just fails. It's basically used to determine container names for running the tests.